torch/onnx/symbolic_opset9.py

```diff
@@ -362,12 +362,18 @@ def _maybe_cast_reduce_op_input(g, self):
     return self
 
 
+def _handle_reduce_dim_none(g, self, op_name):
+    if len(list(self.node().inputs())) > 0:
+        return g.op(op_name, self, keepdims_i=1)
```

Comment:
is `keepdims_i=1` producing same result for empty `self` as pytorch? but not when `keepdims_i=0`? If so please document this behavior in comments

Reply:
Update.